web/src/app/rotations/RotationUsersList.js

```diff
@@ -0,0 +1,262 @@
+import React, { useState } from 'react'
+import p from 'prop-types'
+import gql from 'graphql-tag'
+import FlatList from '../lists/FlatList'
+import Card from '@material-ui/core/Card'
+import CardHeader from '@material-ui/core/CardHeader'
+import OtherActions from '../util/OtherActions'
+import CountDown from '../util/CountDown'
+import RotationSetActiveDialog from './RotationSetActiveDialog'
+import RotationUserDeleteDialog from './RotationUserDeleteDialog'
+import { DateTime } from 'luxon'
+import { UserAvatar } from '../util/avatar'
+import { makeStyles } from '@material-ui/core'
+import { styles as globalStyles } from '../styles/materialStyles'
+import { useMutation } from 'react-apollo'
+import DialogTitleWrapper from '../dialogs/components/DialogTitleWrapper'
+import DialogContentError from '../dialogs/components/DialogContentError'
+import Dialog from '@material-ui/core/Dialog'
+import { query as rotationDetailsQuery } from './RotationDetails'
+
+const updateRotMutation = gql`
+  mutation updateRotation($input: UpdateRotationInput!) {
+    updateRotation(input: $input)
+  }
+`
+
+const useStyles = makeStyles(theme => ({
+  cardHeader: globalStyles(theme).cardHeader,
+}))
+
+export default function RotationUsersList(props) {
```

Comment:
This should be `RotationUserList` (singular as it's a list of User items).

Reply:
I had changed it here with the inverse in my mind (plural as it is a list of users).